get_defs.py

```diff
@@ -5,40 +5,45 @@ def api_call(word: str) -> list:
     
     trial = 0
     time.sleep(0.5)  #0.4 sleep with 2 attempts has had best results
```

Comment:
we'll come back to this later; how to best handle a rate limited api / how its commonly handled